pkg/controller/ibmblockcsi/ibmblockcsi_controller.go

```diff
@@ -283,15 +302,41 @@ func (r *ReconcileIBMBlockCSI) reconcileCSIDriver(instance *ibmblockcsi.IBMBlock
 			return err
 		}
 	} else if err != nil {
-		recLogger.Error(err, "Failed to get CSIDriver", "Name", cd.GetName())
-		return err
+		if r.serverVersion == "1.13" {
+			return r.reconcileCSIDriver113(instance)
+		} else {
+			recLogger.Error(err, "Failed to get CSIDriver", "Name", cd.GetName())
```

Comment:
r u sure here?
why to fail of no 1.13?
iguess if ci pass it means that its ok
but just double check